framework/src/constraints/AutomaticMortarGeneration.C

```diff
@@ -1860,8 +1879,8 @@ AutomaticMortarGeneration::projectPrimaryNodes()
 
 void
 AutomaticMortarGeneration::projectPrimaryNodesSinglePair(
-    subdomain_id_type lower_dimensional_primary_subdomain_id,
-    subdomain_id_type lower_dimensional_secondary_subdomain_id)
+    SubdomainID lower_dimensional_primary_subdomain_id,
```

Comment:
what's the gist between `subdomain_id_type` and `SubdomainID`?

Reply:
they are the exact same, but the former is "libMesh" style while the latter is "MOOSE" style